packages/user/src/services/invite.ts

```diff
@@ -151,9 +151,6 @@ export default class InviteService<
 
   private generateToken(data: any): string {
     const jwtSecret: string = this.getOption("jwt_secret")
-    const expiresIn: number =
```

Comment:
Question: Do we not want to have support for configuring the duration?

Reply:
I initially removed this since we're validating the invite expiration anyways, but I'll re-add it 😄